tests/salesforce/test_salesforce_pipeline.py

```diff
@@ -0,0 +1,46 @@
+from tests.utils import ALL_DESTINATIONS, assert_load_info, load_table_counts
+import pytest
+import dlt
+from pipelines.salesforce import salesforce_source
+
+@pytest.mark.parametrize('destination_name', ALL_DESTINATIONS)
+def test_all_resources(destination_name: str) -> None:
+
+    expected_tables = ['account', 'campaign', 'contact', 'lead', 'opportunity', 'pricebook_2', 'pricebook_entry', 'product_2', 'sf_user', 'user_role']
+
+    pipeline = dlt.pipeline(
+        pipeline_name="salesforce", destination=destination_name, dataset_name="salesforce_data",
```

Comment:
add full_refresh=True. now you load data to the same dataset, there may be race condition on CI